models/image.go

```diff
@@ -43,7 +43,7 @@ type Upload struct {
 type Download struct {
 	Size    int    `bson:"size,omitempty"           json:"size,omitempty"`
 	Href    string `bson:"href,omitempty"           json:"href,omitempty"`
-	Public  string `bson:"public,omitempty"         json:"pubic,omitempty"`
+	Public  string `bson:"public,omitempty"         json:"public,omitempty"`
```

Comment:
🙈

Reply:
I should have identified this funny typo before, hehe